### PR TITLE
(PUP-7042) Mark strings in feature

### PR DESCRIPTION
**Pull request by Magisus**
This commit marks user-facing error and info strings in
`lib/puppet/feature/*` for translation.

### CODE REVIEW

**Comment by Magisus** [comment]
Closing and reopening to fix Travis.

**Comment by Magisus** [comment]
Closing and reopening to fix CI.

File: lib/puppet/feature/base.rb [hunk 1]
```diff
@@ -32,11 +32,11 @@
     require 'win32/service'
     true
   rescue LoadError => err
-    warn "Cannot run on Microsoft Windows without the win32-process, win32-dir and win32-service gems: #{err}" unless Puppet.features.posix?
+    warn _("Cannot run on Microsoft Windows without the win32-process, win32-dir and win32-service gems: %{err}") % { err: err } unless Puppet.features.posix?
```
Comment (geoffnichols) [inline lib/puppet/feature/base.rb]: Would we translate gem names - e.g. "win32-process", "win32-dir", and "win32-service"? (If not, might possibly be worth a translator note?)

Comment (MosesMendoza) [inline lib/puppet/feature/base.rb]: +1 to translator note re:the program names

**Comment by McdonaldSeanp** [comment]
👍

**Comment by Magisus** [comment]
Updated.

**Pull request merged**